ios/Podfile.lock

```diff
@@ -1083,7 +1083,7 @@ SPEC CHECKSUMS:
   pagopa-io-react-native-login-utils: 9fb43fd59dcc864a24343209e74bd7429659456a
   pagopa-react-native-zendesk: 60a26f8a8072234789c34bb7c8a769c156eb57dc
   PromisesObjC: c50d2056b5253dadbd6c2bea79b0674bd5a52fa4
-  RCT-Folly: 424b8c9a7a0b9ab2886ffe9c3b041ef628fd4fb1
+  RCT-Folly: 0080d0a6ebf2577475bda044aa59e2ca1f909cda
```

Comment:
I think the changes to the `ios/Podfile.lock` can be discarded. Try to deintegrate your pods locally and reinstall them. These two hashes were wrongly modified from the PR this one depends on.